README.md

```diff
@@ -31,44 +31,31 @@ Also:
 
 ## Anvil
 
-### Dependencies
+Like others, Smithay as a compositor library has its own sample compositor: anvil.
+
+To get informations about it and how you can run it visit [anvil README](https://github.com/Smithay/smithay/blob/master/anvil/README.md)
+
+## Other compositors that use Smithay
+
+- [Cosmic](https://github.com/pop-os/cosmic-epoch): Next generation Cosmic desktop environment
+- [Catacomb](https://github.com/catacombing/catacomb): A Wayland Mobile Compositor
+- [MagmaWM](https://github.com/MagmaWM/MagmaWM): A versatile and customizable Wayland Compositor
+- [Niri](https://github.com/YaLTeR/niri): A scrollable-tiling Wayland compositor
+- [Strata](https://github.com/StrataWM/strata): A cutting-edge, robust and sleek Wayland compositor
+- [Pinnacle](https://github.com/Ottatop/pinnacle): A WIP Wayland compositor, inspired by AwesomeWM 
+- [Sudbury](https://gitlab.freedesktop.org/bwidawsk/sudbury): Compositor designed for ChromeOS
 
-You'll need to install the following dependencies (note, that those package
-names may vary depending on your OS and linux distribution):
+## System Dependencies
+
+(This list can depend of features you enable)
 
 - `libwayland`
 - `libxkbcommon`
-
-##### These are needed for the "Udev/DRM backend"
 - `libudev`
 - `libinput`
 - `libgbm`
 - [`libseat`](https://git.sr.ht/~kennylevinsen/seatd)
-
-If you want to enable X11 support (to run X11 applications within anvil),
-then you'll need to install the following packages as well:
-    - `xwayland`
-
-### Build and run
-
-Like others, Smithay as a compositor library has its own sample compositor: anvil.
-
-You can run it with cargo after having cloned this repository:
-
-```
-cd anvil;
-
-cargo run -- --{backend}
-```
-
-The currently available backends are:
-
-- `--x11`: start anvil as an X11 client. This allows you to run the compositor inside an X11 session or any compositor supporting XWayland. Should be preferred over the winit backend where possible.
-- `--winit`: start anvil as a [Winit](https://github.com/tomaka/winit) application. This allows you to run it
-  inside of an other X11 or Wayland session.
-- `--tty-udev`: start anvil in a tty with udev support. This is the "traditional" launch of a Wayland
-  compositor. Note that this requires you to start anvil as root if your system does not have logind
-  available.
+- `xwayland`
 
 ## Contact us
 
```

Comment:
Maybe we can remove the mention of the IRC chatroom while we're at it (just below here)

Reply:
Done